### PR TITLE
[Feat] update runtime to v22.08

### PR DESCRIPTION
**Pull request by flavioislima**
Implements https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/1771

### CODE REVIEW

**Comment by farribeiro** [comment]
needs extensions be published with 22.08 runtime

**Comment by flavioislima** [comment]
> needs baseapp and extensions be published with 22.08 runtime

wdym? on the flatpak side?

**Comment by farribeiro** [comment]
most of extensions stay building last days to 22.08 and not done yet

**Comment by farribeiro** [comment]
bot, build

**Comment by JakobDev** [comment]
`org.electronjs.Electron2.BaseApp` is now available for 22.08

**Comment by flavioislima** [comment]
> `org.electronjs.Electron2.BaseApp` is now available for 22.08

Since we are extracting the appImage, do we need the electron runtime?
I removed it on this PR.

**Comment by flavioislima** [comment]
> 

Do you know which extensions are working and which one are not? which commits should be reverted?

**Comment by JakobDev** [comment]
Looks like the builderror happens on vulakn-tools. You should try to update vulkan-tools in the manifest to the latest version.

**Comment by flavioislima** [comment]
> Looks like the builderror happens on vulakn-tools. You should try to update vulkan-tools in the manifest to the latest version.

yes, this was my first try and it didn't fixed it.

**Comment by JakobDev** [comment]
@flavioislima I managed to get vulkan-tools to build. I'm currently testing a few things, before I post the new manifest. I hope there will be no error in the other modules.

btw:
Why did you build Perl? It is already included in the Freedesktop SDK.

**Comment by JakobDev** [comment]
I have made a new PR

**Comment by flavioislima** [comment]
Thanks we need Perl because winetricks can't find Perl inside the flatpak. At least on 21.08. Haven't tested with 22.08 since we just merged the pr.

**Comment by JakobDev** [comment]
I'm surprised that Winetricks need Perl at all. I didn't [find anything matching Perl outside a comment.](https://github.com/Winetricks/winetricks/search?q=perl). This should be reported to the developers.

How can I test if Winetricks can find Perl? Perl takes a lot time to build, compared to the other Modules, so it makes sense to try to remove it.

File: com.heroicgameslauncher.hgl.yml [hunk 1]
```diff
@@ -1,17 +1,14 @@
 id: com.heroicgameslauncher.hgl
 sdk: org.freedesktop.Sdk
 runtime: org.freedesktop.Platform
-runtime-version: '21.08'
-base: org.electronjs.Electron2.BaseApp
-base-version: '21.08'
```
Comment (TheEvilSkeleton) [inline com.heroicgameslauncher.hgl.yml:6]: I thought HGL uses Electron?

**Comment by flavioislima** [comment]
@JakobDev I think if perl was not found most of commands in winetricks wont work
@Nocccer found this issue some time ago and added it to fix that.
But yes, compiling perl is a pain just because of winetricks.

I am closing this but lets keep talking.

**Pull request closed without merging**